app/controllers/hyrax/stats_controller.rb

```diff
@@ -35,20 +35,32 @@ def add_breadcrumb_for_action
       end
     end
 
-    def date_range
-      start_from = case params[:start_from].to_s
-        when '1_month'
-          Time.zone.today - 1.month
-        when '3_month'
-          Time.zone.today - 3.month
-        when '6_month'
-          Time.zone.today - 6.month
-        when '1_year'
-          Time.zone.today - 1.year
-        else
-          Hyrax.config.analytics_start_date
-        end
-      "#{start_from},#{Time.zone.today}" 
+    def date_range_for_download_statistics
+      "#{params[:start_date] || Hyrax.config.analytics_start_date},#{params[:end_date] || Date.today}" 
```

Comment:
Do we not need to check the start date is greater than or equal to Hyrax.config.analytics_start_date